swarm/src/behaviour.rs

```diff
@@ -340,6 +327,13 @@ pub trait PollParameters {
 ///
 /// You can opt out of this behaviour through `#[behaviour(event_process = false)]`. See the
 /// documentation of [`NetworkBehaviour`] for details.
+#[deprecated(
+    since = "0.38.0",
+    note = "Use `#[behaviour(out_event = \"MyBehaviourEvent\")]` instead. See \
+            https://github.com/libp2p/rust-libp2p/blob/master/swarm/CHANGELOG.md#0380 \
```

Comment:
I think it would be better to link to a heading rather than a line. This line will change as we add more entries to it because we add to the top, meaning the reference will point to the wrong thing.

Reply:
`#0380` is the header once we remove `[unreleased]` from it. Right now it is:

```
https://github.com/libp2p/rust-libp2p/blob/master/swarm/CHANGELOG.md#0380-unreleased
```

Does that make sense @thomaseizinger?

Reply:
Ah yes, I thought this was the line number.